internal/pkg/version/protocol_versions.go

```diff
@@ -13,6 +13,9 @@ const DEVNET4 = "devnet4"
 // LOCALNET is the network name of localnet
 const LOCALNET = "localnet"
 
+// TESTNET3 is the network name of the third testnet
+const TESTNET3 = "localnet-270a8688-1b23-4508-b675-444cb1e6f05d"
```

Comment:
Required to interop with testnet genesis file. This will be fixed in a subsequent PR.